enhancements/sig-architecture/179-work-feedback-scrape-type/README.md

```diff
@@ -0,0 +1,150 @@
+more dynamic, scalable, and adaptive to workload characteristics.
+
+### Goals
+
+* Provide configurable feedback collection modes (POLL or WATCH) per resource in ManifestWork. 
+
+* Ensure MWRS correctly aggregates and reflects feedback updates from both polling and watch-based sources.
+
+* Maintain full backward compatibility for existing users and automation.
+
+### Non-Goals
+
+* Changing the core aggregation semantics in MWRS beyond supporting the new event-driven updates.
+
+## Proposal
+
+This is where we get down to the nitty gritty of what the proposal actually is.
+
+### User Stories
+
+Detail the things that people will be able to do if this is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+
+#### Story 1
+
+User A uses MWRS to deploy a rollout to multiple cluster. The user wants 
+to track the number of pod replicas that contain the change as it is rolled out to the cluster on the MWRS level.
+To do this, the user must get the replica count as the change is rolling out, if it relies on the existing mechanism, it is likely that the change will be missed. 
+
+For example, if the rollout happens every 10 seconds, then User A will not be able to track the per replica count. And only get the count when the rollout is completed.
+
+### Implementation Details/Notes/Constraints [optional]
+
+What are the caveats to the implementation? What are some important details that
+didn't come across above. Go in to as much detail as necessary here. This might
+be a good place to talk about core concepts and how they relate.
+
+### Risks and Mitigation
+
+What are the risks of this proposal and how do we mitigate. Think broadly. For
+example, consider both security and how this will impact the larger OKD
+ecosystem.
+
+How will security be reviewed and by whom? How will UX be reviewed and by whom?
+
+Consider including folks that also work outside your immediate sub-project.
+
+## Design Details
+
+#### ManifestWork CRD Changes 
+```
+apiVersion: work.open-cluster-management.io/v1alpha1
+kind: ManifestWorkReplicaSet
+...
+spec:
+  workload: ...
+  manifestConfigs:
+    - resourceIdentifier:
+        group: apps
+        resource: deployments
+        namespace: default
+        name: hello
+      feedbackScrapeType: WATCH (Options: WATCH | POLL, POLL is default behavior)
+    ....
+```
+
+This change will also be populated upwards to the ManifestWorkReplicaSet level of 
+`spec.manifestWorkTemplate.manifestConfigs`. 
+
+#### Spoke Level ManifestWork Controller Reconciler 
+
+Default: resources continue to be polled on the existing interval.
+If a resource is marked WATCH, the controller registers it for watch-based updates:
+Immediate status/feedback recomputation on events.
+Poll remains as a safety net.
+
+Create a FeedbackScrape interface that polls or watches resources. 
```

Comment:
will this only impact resource status feedback, or it also impacts resource apply logic?

Reply:
Which apply logic?

Reply:
oh I mean this controller https://github.com/open-cluster-management-io/ocm/blob/main/pkg/work/spoke/controllers/manifestcontroller/manifestwork_controller.go